terraform-runner/Dockerfile

```diff
@@ -32,7 +32,7 @@ RUN curl -L https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cl
 ENV PATH="${PATH}:/google-cloud-sdk/bin"
 
 # install some random stuff
-RUN apt-get install -y jq
+RUN apt-get install -y jq vim nano
```

Comment:
Doesn't hurt, but I've found it a lot easier to edit the files *outside* the container, and just run the container in another terminal.